src/main/java/teammates/ui/webapi/GetFeedbackSessionLogsAction.java

```diff
@@ -45,6 +45,12 @@ void checkSpecificAccessControl() {
 
         InstructorAttributes instructor = logic.getInstructorForGoogleId(courseId, userInfo.getId());
         gateKeeper.verifyAccessible(instructor, courseAttributes);
```

Comment:
There is an overloaded method for the gatekeeper that accepts permission so you don't need to write the code below